apps/src/weblab2/Weblab2View.tsx

```diff
@@ -83,7 +83,7 @@ const defaultProject: ProjectType = {
   <link rel="stylesheet" href="styles.css"/>
   <body>
     Content goes here!
-    <div class="foo">Foo class!</div>
+    <div class="foo">[DEFAULT] Foo class!</div>
```

Comment:
Just to hopefully help us see when we're loading in default data. I'd encourage everyone to change your data when you navigate in and save, so that if this ever shows up it's more clear.